libenv/sysinfo.c

```diff
@@ -1058,6 +1058,51 @@ static void SetFlavour(EvalContext *ctx, const char *flavour)
 }
 
 #ifdef __linux__
+
+// NOTE: strings should be canonified before calling
+static void SetFlavour2(
+    EvalContext *const ctx,
+    const char *const id,
+    const char *const major_version)
+{
+    assert(ctx != NULL);
+    assert(id != NULL);
+    assert(major_version != NULL);
+
+    char *flavor;
```

Comment:
Please stick to the *flavour* spelling everywhere.

Reply:
Yes it's a little annoying, but I would argue that we should default to american spelling. `sys.flavor` is much more used in our codebase, this file is pretty much the only culprit when it comes to using british spelling.

Reply:
I changed all occurences to flavor instead.